Article/Article.js

```diff
@@ -112,3 +144,67 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new artible
 
 */
+
+const articles = document.querySelector(".articles");
+
+data.forEach(data => {
+  console.log(data.title);
+  articles.appendChild(
+    create(
+      data.title,
+      data.date,
+      data.firstParagraph,
+      data.secondParagraph,
+      data.thirdParagraph
+    )
+  );
+});
+
+////create elements
+function create(title, date, first, second, third) {
+  const articles = document.createElement("div");
+  const article = document.createElement("div");
+  const articleTitle = document.createElement("h2");
+  const articleDate = document.createElement("p");
+  const firstPara = document.createElement("p");
+  const secondPara = document.createElement("p");
+  const thirdPara = document.createElement("p");
+  const expandButton = document.createElement("span");
+
+  console.log(articleDate);
```

Comment:
I love seeing the use of console.log's for debugging. When I know I am done with my project, I try to go back and remove all of them in a commit so that the codebase is 'clean' when it gets merged into 'production'.